src/Logger/Logger.cc

```diff
@@ -55,6 +57,10 @@ void InitLogger(bool no_log_file, int verbosity) {
         case 6:
             stdout_logger->set_level(spdlog::level::trace);
             break;
+        case 7:
+            stdout_logger->set_level(spdlog::level::trace);
```

Comment:
This should be `debug` now, and we should switch 6 and 7 around.

Reply:
I don't know about that actually. The ICD logging is the most verbose logging we'd want, so it should still be 7.